src/pattern-library/components/patterns/ColorFoundations.js

```diff
@@ -1,20 +1,16 @@
-import {
-  PatternPage,
-  Pattern,
-  PatternExamples,
-  PatternExample,
-} from '../PatternPage';
+import Library from '../Library';
 
 // TODO:
 // - color descriptions and guidance
 // - hex and other metadata about colors
-// - more sophisticated "swatches"
 // - foreground color examples
 // - more examples of overriding with `!` classes
 // - valid contrast combinations
 
-const backgroundExamples = [
-  'brand',
+const brand = ['brand'];
+
+const greys = [
```

Comment:
Pedantic:
```suggestion
const monochromeSpectrum = [
```

Reply:
Worth a discussion, sure! I don't want to roll it into this changeset, as these changes are mechanics, and what you're suggesting is a conceptual change.